src/components/NurseryWithdrawals/NurseryReassignment.tsx

```diff
@@ -222,7 +222,7 @@ export default function NurseryReassignment(): JSX.Element {
         <Box>
           <BackToLink
             id='back'
-            to={APP_PATHS.NURSERY_WITHDRAWALS}
+            to={`${APP_PATHS.NURSERY_WITHDRAWALS}?tab=withdrawal_history`}
```

Comment:
Should we add it only if tracking enabled? I think it doesn't break adding it though